setup.py

```diff
@@ -4,7 +4,6 @@
 # Full license can be found in License.md
 # -----------------------------------------------------------------------------
 
-import sys
 import os
 from codecs import open
```

Comment:
Recommend not overwriting existing function `open`

Reply:
Updated.  Since I pulled this format from `pysat`, I've initiated changes there too for consistency.